scripts/set-latest-release.ts

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env ts-node
+
+// This script updates `builds.radicle.xyz/latest.json` with the
+// current.
+//
+// This script runs `gsutil` so you need to be logged into an account
+// that has permissions to write to the `builds.radicle.xyz` bucket.
+
+const fetch = require("node-fetch");
+const path = require("path");
+const fs = require("fs");
+const childProcess = require("child_process");
+
+const pkg = require("../package.json");
+
+const fileName = "latest.json";
```

Comment:
Should we add this to `.gitignore`?

Reply:
This file is created in a temporary directory so we don’t need to add it to `.gitignore`.